source/localizable/routing/defining-your-routes.md

```diff
@@ -250,16 +249,23 @@ In the next section, [Specifying a Route's Model](../specifying-a-routes-model),
 
 ## Wildcard / globbing routes
 
-You can define wildcard routes that will match multiple URL segments. This could be used, for example,
-if you'd like a catch-all route which is useful when the user enters an incorrect URL not managed
-by your app.
+You can define wildcard routes that will match multiple URL segments.
+This could be used, for example, if you'd like a catch-all route which is useful when the user enters an incorrect URL not managed by your app.
+Wildcard routes begin with an asterisk.
 
 ```app/router.js
 Router.map(function() {
-  this.route('page-not-found', { path: '/*wildcard' });
```

Comment:
Changed from `page-not-found` to `not-found` because that's what I found a lot. I changed `*wildcard` to `*path` for 2 reasons:  
1. It's a wildcard path, and I thought `*path` reads as such i.e. `*` means wildcard.
2. I didn't want any possible confusion where learners took the wrong message that all wildcard routes should be called `*wildcard`. I guess this could happen with `*path` as well but at least it avoids the wildcard keyword.